src/widget/geo/geopicker.js

```diff
@@ -841,6 +840,16 @@ class Geopicker extends Widget {
                 }
             });
 
+            this.map.on('load', () => {
+                this.map.on('zoomend', (event) => {
```

Comment:
Note that this is wrapped in the `load` event to ensure the initial zoom isn't treated as a user-initiated state change.